include/kframework/evm.md

```diff
@@ -1423,10 +1425,10 @@ The various `CALL*` (and other inter-contract control flow) operations will be d
          <output> OUT </output>
          <gas> GAVAIL </gas>
 
-    syntax InternalOp ::= "#refund" Exp [strict]
+    syntax InternalOp ::= "#refund" Exp
```

Comment:
@ehildenb 
Not sure if the update to `K` caused this, but `#refund` was running into an error where `~> #freezer#refund__EVM_InternalOp_Exp0_ ( )` was left on the `<k>` cell when refunding a `#gas`. The crashing tests were passing locally, but unsure if there are other issues or efficiency costs caused by removing `[strict]` though.

Reply:
I am surprised it works at all with removing `strict` here actually, as far as I know, we definitely need that there....

Reply:
Actually, looking at it, it seems that we probably don't need taht to be `strict`. But if that's so, we should lower teh sort from `Exp` to `Gas` instead (of the argument).

```suggestion
    syntax InternalOp ::= "#refund" Gas
```